### DISCUSSION TITLE
Add padding to the right of the new Big Sur dock.

### DISCUSSION THREAD

**Discussion by psande** [rxhanson/Rectangle#296] [Ideas]
It would be nice to have an option that adds padding to the right of the new dock, same as it has on the left side.

[Screenshot for reference](https://imgur.com/YTASJwU)

Hope it makes sense! And thanks for your work!

**Comment by rxhanson** [comment]
Right, I see what you mean. I'll keep this in mind on a future release. I'd also consider a pull request and point anyone in the right direction if they're interested in tackling it.

**Reply by psande** [reply]
As a developer I would love to make a PR and help out but unfortunately I don't have experience in Apple development. I will take a look out of curiosity, but I doubt I will be able to do it.

Thanks anyway for keeping it in mind for future releases.

Pablo
